src/integrationTest/java/org/opensearch/test/framework/audit/AuditMessagePredicate.java

```diff
@@ -60,10 +62,11 @@ private AuditMessagePredicate(
         this.transportRequestType = transportRequestType;
         this.effectiveUser = effectiveUser;
         this.index = index;
+        this.privilege = privilege;
```

Comment:
What is the privilege field for here?

Reply:
It is the action String: i.e. "cluster:admin/..."